### DISCUSSION TITLE
Make a Snip button

### DISCUSSION THREAD

**Discussion by gazalied** [ScratchAddons/ScratchAddons#1487] [Ideas]
Make snipping much easier!
![Screenshot 2021-02-02 200741](https://user-images.githubusercontent.com/76770572/106604890-d4844180-6592-11eb-8a06-f1ec4f96feed.png)
![Screenshot 2021-02-02 201025](https://user-images.githubusercontent.com/76770572/106604919-de0da980-6592-11eb-88b3-22192a89b0bb.png)

**Comment by TheColaber** [comment]
This has basically been done in an addon called Better forum quoter
![image](https://user-images.githubusercontent.com/72760579/106615946-fa7df780-653a-11eb-95fb-648d2f7dbf7c.png)

**Reply by gazalied** [reply]
But it will just "only quote the selected text". I want it to just say "-snip-" so I don' think that's a workaround.

**Comment by cobaltt7** [comment]
That is a feature that could be easily added.
________________________________

**Reply by cobaltt7** [reply]
I will do this

**Reply by cobaltt7** [reply]
Draft: https://github.com/ScratchAddons/ScratchAddons/compare/master...RedGuy12:patch-1 todo: don't add it if no text before/after

**Reply by gazalied** [reply]
> I will do this

Sorry for the late reply but Thanks!

**Reply by GrahamSH-LLK** [reply]
> Draft: [master...RedGuy12:patch-1](https://github.com/ScratchAddons/ScratchAddons/compare/master...RedGuy12:patch-1) todo: don't add it if no text before/after

The json is invalid.

**Reply by cobaltt7** [reply]
Thanks for letting me know @GrahamSH-LLK. I fixed it.

**Comment by gazalied** [comment]
bump?
